### PR TITLE
Added context block to Describe

### LINKED ISSUE #21: add Context
so it can be used like:
```ruby
BDDSM.describe '.context' do
  let(:user) { create(:user) }
 
  context 'when user is valid' do
    it { expected(user).to be valid }
  end
 
   context 'when user is invalid' do
     let(:user) { User.new }
   
     it { expected(user).to be invalid }
   end
end
```

### PR DESCRIPTION
**Pull request by fartem**

### CODE REVIEW

**Comment by fartem** [comment]
Closed #21.

File: test/describe_context.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+BDDSM.describe 'Test case with context' do
+  context 'Context example 1' do
```
Comment (sergio-fry) [inline test/describe_context.rb:2]: Is it possible to add tests to check error messages generated when such test failed?

Comment (sergio-fry) [inline test/describe_context.rb:2]: Is it possible to have nested contexts?

**Review by sergio-fry — CHANGES_REQUESTED** [review]

File: test/describe_context.rb (hunk 1 above)
Comment (fartem) [reply test/describe_context.rb:2]: Not sure about you are meaning. Maybe you can give me some example of this?

Comment (fartem) [reply test/describe_context.rb:2]: Yes. I added one example to tests few seconds ago.

**Review by sergio-fry — APPROVED** [review]

**Pull request merged**